src/components/shoppingLists.js

```diff
@@ -25,13 +26,14 @@ export class ShoppingLists extends Component {
             onClick={() => this.clickedAList(list.name)}
             to={`/lists/${list.id}`}
           >
-            Name: {list.name} Address: {list.address}
+            {list.name}
+            <br /> {list.address}
```

Comment:
I think this should be `list.store.address`

Reply:
that is correct and fixed